build/Linux+BSD/mscore.1.in

```diff
@@ -1,244 +1,531 @@
-.\" .ad b      justify to both left and right margins
-.\" .nf        disable filling
-.\" .fi        enable filling
-.\" .br        insert line break
-.\" .sp <n>    insert n+1 empty lines
-.\" for manpage-specific macros, see man(7)
-.\" TeX users may be more comfortable with the \fB<whatever>\fP and
-.\" \fI<whatever>\fP escape sequences to invode bold face and italics,
-.\" respectively.
-.SH NAME@Variables_substituted_by_CMAKE_on_installation@
-mscore@MSCORE_INSTALL_SUFFIX@, musescore@MSCORE_INSTALL_SUFFIX@ \- @MUSESCORE_NAME_VERSION@ sheet music editor.
-
-.SH SYNOPSIS
-@BEGIN_section_to_only_appear_in_portable_builds@
-@MAN_PORTABLE@.SS Before installation:
-@MAN_PORTABLE@.B ./@PORTABLE_INSTALL_NAME@.AppImage
-@MAN_PORTABLE@.RI [ OPTIONS ]
-@MAN_PORTABLE@.RI [ FILE .\|.\|.]
-
-@MAN_PORTABLE@.SS After installation:
-@END_section_to_only_appear_in_portable_builds@
-.B mscore@MSCORE_INSTALL_SUFFIX@
-.RI [ OPTIONS ]
-.RI [ FILE .\|.\|.]
-.br
-.B musescore@MSCORE_INSTALL_SUFFIX@
-.RI [ OPTIONS ]
-.RI [ FILE .\|.\|.]
-
-.SH DESCRIPTION
-MuseScore is a free and open source WYSIWYG program for typesetting
-musical scores, released under the GNU General Public Licence (GPLv2).
-
-@BEGIN_section_to_only_appear_in_portable_builds@
-@MAN_PORTABLE@.SS Portable version
-@MAN_PORTABLE@This portable version of MuseScore has all of MuseScore's usual features,
-@MAN_PORTABLE@but it runs on a wider range of distributions and does not need to be
-@MAN_PORTABLE@installed. There is an option to install it for full integration with other
-@MAN_PORTABLE@applications and the desktop environment.
-@END_section_to_only_appear_in_portable_builds@
-
-.SS Getting help
-This manual documents command line usage of
-.BR mscore@MSCORE_INSTALL_SUFFIX@ .
-For help with the full MuseScore program see:
-.RS
-.TP
-.B Online Handbook
-<https://musescore.org/handbook>
-.TP
-.B Support Forum
-<https://musescore.org/forum>
-.RE
-
-.SS Further information
-.TP
-These pages cover the topics in this manual and may be more up-to-date:
-
-<https://musescore.org/handbook/command\-line\-options\-0>
-.br
-<https://musescore.org/handbook/revert\-factory\-settings\-0>
-
-.SH OPTIONS
-A summary of options is included below. Running
-.B mscore@MSCORE_INSTALL_SUFFIX@
-without options launches the full MuseScore
-program and opens any specified files.
-
-@BEGIN_section_to_only_appear_in_portable_builds@
-@MAN_PORTABLE@.SS Special options for the portable version
-@MAN_PORTABLE@.TP
-@MAN_PORTABLE@.B \-h, \-\-help
-@MAN_PORTABLE@Lists the various command line options and their usage.
-@MAN_PORTABLE@.TP
-@MAN_PORTABLE@.B man, manual, manpage
+.\" New manual page for MuseScore 3.0
+.\" Copyright (c) 2018
+.\"	mirabilos <m@mirbsd.org>
+.\" Published under the same terms as MuseScore itself.
+.\"-
+.Dd December 18, 2018
+.Dt @MAN_MSCORE_UPPER@ 1
+.Os MuseScore
+.Sh NAME@Variables_substituted_by_CMAKE_on_installation@
+.Nm mscore@MSCORE_INSTALL_SUFFIX@ ,
+.Nm musescore@MSCORE_INSTALL_SUFFIX@
```

Comment:
MuseScore3

Reply:
These are commands used to launch MuseScore from the command line, so they have to be lowercase, and there is no 3 in the executable name on Linux.

Reply:
in Linux and Mac it is `mscore`, lowercase, on Windows it is MuseScore3, camel case (but case insitive) and with the 3.

Reply:
`mscore` is on the line above

Reply:
No, not here; `.Nm` is the name of the binaries, and the two listed are correct and double-checked.

Reply:
No, it is `MuseScore3.exe` for MuseScore 3, `MuseScore.exe` for MuseScore 2

Reply:
No, it is `MuseScore3.exe` for MuseScore 3, `MuseScore.exe` for MuseScore 2

Reply:
This is the Linux manual and the Linux binaries are name `mscore` and `musescore`.

Reply:
This is the Linux **and Mac** manual, and to the best of my knowledge there is it called `mscore', not `musescore`, but in order to also cover the Windows part it could/should mentions `MuseScore3.exe`

Reply:
Unfortunately, this manual is not available on macOS. It *might* be possible to make it available somehow via Homebrew, but I don't know of any casks (GUI Homebrew apps) that provide manuals, so almost nobody would find it.

Windows cannot use this manual as there is no viewer for `troff` files built in on Windows. A manual page could be generated via troff to pdf conversion, but I think this is beyond the scope of this PR.